modules/cnv-template-vmi-pxe-config.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
```

Comment:
Something is invalid with this template, but I still haven't figured out what.
When deploying it, after creating the NetworkAttachmentDefinition from cnv-pxe-booting-with-mac-address.adoc, the VMI creation fails with this error:
`  Warning  SyncFailed        0s (x5 over 0s)  virt-handler, working-pvsnt-worker-0-j84d6  server error. command SyncVMI failed: "preparing the pod network failed: Link not found"`
I tried another template, which uses the same NetworkAttachmentDefinition, and it works fine.
The one I used is [here](http://pastebin.test.redhat.com/759323), and I created it by running the command:
`oc process -f vm-template-fedora-multus-mac.yaml -p NAME=cnv-127-vm -p MULTUS_NETWORK=pxe-net-conf | oc create -f -`

Reply:
Hey, thanks for the heads up. I'm not sure what changes to make here, and I'm not familiar with using that command to create a template. I know that we need the template to use `masquerade` so at the very least, that would need to be changed from the one you provided...

@dankenigsberg can you weigh in on this?

Reply:
Sorry for the lack of response here. We were trying to understand the bug.

Per Sebastian, it is a bug in kubemacpool.

Please include the template as it is. If we have a bug there, we'd fix it.

Let's have this pr merged promptly.